src/components/user-group/repo/classes.repo.integration.test.js

```diff
@@ -0,0 +1,129 @@
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const appPromise = require('../../../app');
+const testObjects = require('../../../../test/services/helpers/testObjects')(appPromise);
+const { classesRepo } = require('.');
+
+chai.use(chaiAsPromised);
+const { expect } = chai;
+
+describe('class repo', () => {
+	let app;
+	let server;
+
+	before(async () => {
+		app = await appPromise;
+		server = await app.listen(0);
+	});
+
+	afterEach(async () => {
+		await testObjects.cleanup();
+	});
+
+	after(async () => {
+		await server.close();
+	});
+
+	describe('when get classes for user', async () => {
+		it('should return list of classes where the user is part of ', async () => {
+			const { _id: schoolId } = await testObjects.createTestSchool();
+			const user = await testObjects.createTestUser({ roles: ['student', 'teacher'], schoolId });
+			const testClass = await testObjects.createTestClass({ userIds: [user._id], teacherIds: [user._id], schoolId });
+			const classes = await classesRepo.getClassesForUser(user._id);
+
+			expect(classes.length, 'only one class expected').to.be.equal(1);
+			expect(classes[0]._id).to.deep.equal(testClass._id);
```

Comment:
would prefer objectId equal helper here

Reply:
.toString() and both position will make it more stable. But objectId.equal and .to.be.true will catch informations